xrmocap/ops/triangulation/fourdag_triangulator.py

```diff
@@ -0,0 +1,564 @@
+import math
```

Comment:
It does not looks like a triangulator, but a person-wise keypoints3d optimizer that needs camera parameters, and it performs triangulation and projection inside it. We had better try to split this file into 2 or more parts, one is the basic triangulation, which is not perfect, but gives us a resonable 3D result. The others are optimizations, and each one has its own name, purpose, docstring. See https://github.com/openxrlab/xrmocap/blob/main/xrmocap/transform/keypoints3d/optim/smpl_shape_aware_optimizer.py for example. This one should be complicated enough.